doc/WebRTC.xml

```diff
@@ -0,0 +1,922 @@
+    <para>Since the headers of the messages differ depending on whether the message is sent between
+      client and server, or between device and server, we specify two separate versions of the
+      interface. One between client and server, called <emphasis>client-facing</emphasis> from now
+      on, and one between device and server, called <emphasis>device-facing</emphasis> from now
+      on.</para>
+    <figure xml:id="_Ref493258797">
+      <title>Signaling flow sequence diagram</title>
+      <mediaobject>
+        <imageobject>
+          <imagedata fileref="media/WebRTC/webrtc_signaling_flow.svg" contentwidth="160mm"/>
+        </imageobject>
+      </mediaobject>
+    </figure>
+    <section xml:id="section_ayt_v12_v5b">
+      <title><?oxy_comment_start author="fredriks" timestamp="20230105T155830+0100" comment="Split this in two under-sections for client and device "?>Connecting
+        to the signaling server</title>
+      <para>Both a client and a device need to connect to the server by setting up a WebSocket with
+        valid access tokens that the server will accept. The connection handling is done using
+        WebSocket [RFC 6455].</para>
+      <para>The signaling messages are then sent as JSON text messages in the WebSocket. Please
+        refer to the Signaling Protocol AsyncAPI schema for specific details regarding the Signaling
+        Protocol.</para><?oxy_comment_end?>
+      <section>
+        <title>Client</title>
+        <para>A client connects to the server by setting up a WebSocket to, for example:
+          <programlisting>https://signaling.example.com/client?authorization=Bearer%20access_token</programlisting></para>
+        <para>Where <code>signaling.example.com</code> should be replaced with the server's address,
+          and <code>access_token</code> should be replaced with a valid
+          <?oxy_comment_start author="fredriks" timestamp="20230105T145309+0100" comment="Add info on how to generate access token"?>access
+          token that the server will accept<?oxy_comment_end?>.</para>
+        <para>See section <xref linkend="chapter_cwp_r2k_v5b"/> for how to get an access
+          token.</para>
+        <para>The WebSocket sub protocol shall be set to 'client.webrtc.onvif.org' for the
+          client.</para>
+      </section>
+      <section>
+        <title>Device</title>
+        <para>A device must first be onboarded in order to be able to set up a WebSocket to the
+          server. See &lt;??> for how to onboard a device.</para>
+        <para>An onboarded device then connects by setting up a WebSocket to, for example:</para>
+        <programlisting>wss://signaling.example.com/device? deviceId=myDeviceId&amp;orgId=myOrgId&amp;token=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjI4NjYzNjg1NDgsInRhcmdldElkIjoiTXlUZXN0VGFyZ</programlisting>
+        <para>Where <code>signaling.example.com</code> should be replaced with the server's address
```

Comment:
"replaced with the signalling server's address" Suggest a find and replace here in the document.